data/chests.py

```diff
@@ -105,6 +106,27 @@ def shuffle_random(self):
             elif chest.type == Chest.ITEM:
                 chest.contents = self.items.get_random()
 
+        if self.args.no_trash_chests:
+            if self.args.shop_sell_fraction4:
+                sell_factor = 1/4
+            elif self.args.shop_sell_fraction8:
+                sell_factor = 1 / 8
+            elif self.args.shop_sell_fraction0:
+                sell_factor = 0
+            else:
+                sell_factor = 1/2
+            for chest in possible_chests:
+                if not chest.type == Chest.ITEM:
+                    continue
+                item = Item(chest.contents, self.rom)
```

Comment:
Please change to `item = self.items.items[chest.contents]` (otherwise there's a merge conflict with other Item changes)